src/rest/Facet/FacetRequest.ts

```diff
@@ -41,6 +45,22 @@ export interface IFacetRequestValue extends IRangeValue {
    * **Default:** `false`
    */
   preventAutoSelect?: boolean;
+  /**
+   * Whether to retrieve the children of this hierarchical facet value. Can only be used on leaf values.
+   * 
+   * **Default:** `false`
+   */
+  retrieveChildren?: boolean;
+  /**
+   * The maximum number of children to retrieve for this category facet value. Ignored if `retrieveChildren` is `false`.
```

Comment:
Maybe we could turn this around to make it clearer? E.g.,

> If [`retrieveChildren`]{@link IFacetRequestValue.retrieveChildren} is `true`, the maximum number of children to retrieve for this leaf value.

Reply:
Also, I believe the default value enforced by the Search API is `0`; perhaps we should document that.

This makes me wonder why we even need the `retrieveChildren` parameter, though. Isn't a default `retrieveCount` value of `0` pretty much equivalent to a default `retrieveChildren` value of `false`?

Reply:
That change would have to be reflected on the API side right because true is required.

Reply:
After discussion with API, we'll keep them separate